src/nunit-gui/Presenters/MainPresenter.cs

```diff
@@ -168,23 +195,50 @@ private void MainForm_Closing(object sender, FormClosingEventArgs ea)
             }
         }
 
-        #region Command Handlers
+        private void SelectedRuntime_SelectionChanged()
+        {
+            ChangePackageSetting(EnginePackageSettings.RuntimeFramework, _view.SelectedRuntime.SelectedItem);
+        }
 
-        private void OnOpenProjectCommand()
+        private void ProcessModel_SelectionChanged()
         {
-            string[] files = _view.DialogManager.GetFilesToOpen();
-            if (files.Length > 0)
-                _model.LoadTests(files);
+            ChangePackageSetting(EnginePackageSettings.ProcessModel, _view.ProcessModel.SelectedItem);
         }
 
-        private void OnReloadTestsCommand()
+        private void DomainUsage_SelectionChanged()
         {
-           _model.ReloadTests();
+            ChangePackageSetting(EnginePackageSettings.DomainUsage, _view.DomainUsage.SelectedItem);
         }
 
-        private void OnReloadTestsCommand(string runtime)
+        private void LoadAsX86_CheckedChanged()
         {
-            _model.ReloadTests(runtime);
+            var key = EnginePackageSettings.RunAsX86;
+            if (_view.RunAsX86.Checked)
+                ChangePackageSetting(key, true);
+            else
+                ChangePackageSetting(key, null);
```

Comment:
I prefer not to clutter the settings, so I want to remove it rather than setting it false. In many cases, the framework interprets the absence of a setting in a different way from __any__ of the accepted values, and ChangePackageSettings is intended to work with as little knowledge of the framework as possible. I kept the "magic" value "DEFAULT" from my earlier work, but it might be better to replace that with null in the calling method as well. Also, in other kinds of settings within NUnit, there is a convention that null means delete it.